crates/topos-p2p/src/runtime/mod.rs

```diff
@@ -38,32 +27,20 @@ pub struct Runtime {
 
     /// Shutdown signal receiver from the client
     pub(crate) shutdown: mpsc::Receiver<oneshot::Sender<()>>,
+
+    pub(crate) current_bootstrap_id: Option<QueryId>,
 }
 
 mod handle_command;
 mod handle_event;
 
 impl Runtime {
-    pub async fn bootstrap(mut self) -> Result<Self, Box<dyn std::error::Error>> {
-        if self.bootstrapped {
-            return Err(Box::new(P2PError::BootstrapError(
-                "Network is already bootstrapped or in bootstrap",
-            )));
-        }
-
-        self.bootstrapped = true;
-
+    pub async fn bootstrap(&mut self) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Missing docs. Maybe link to https://docs.rs/libp2p-kad/latest/libp2p_kad/struct.Behaviour.html#method.bootstrap here? Seems like that provides a good description of the bootstrapping process and explains why things are named like they are?